packages/fhir-converter/src/templates/cda/Sections/Notes.hbs

```diff
@@ -1,12 +1,34 @@
 {{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.65')}}
     {{#with (getAllCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.65') as |sections|}}
+            {{!-- sections here is already an array so this is not a bug --}}
             {{#each sections}}
                 {{#each (toArray 2_16_840_1_113883_10_20_22_2_65.entry) as |note|}}
                     {{>Resources/DiagnosticReport.hbs diagReport=note.act ID=(generateUUID (toJsonString note.act))}},
                     {{#with (evaluate 'Utils/GeneratePatientId.hbs' obj=@metriportPatientId) as |patientId|}}
                         {{>References/DiagnosticReport/subject.hbs ID=(generateUUID (toJsonString note.act)) REF=(concat 'Patient/' patientId.Id)}},
                     {{/with}}
 
+                    {{!-- TODO need to add references, author, etc --}}
+                    {{#if note.act.author.assignedAuthor}}
+                        {{#with (evaluate 'Utils/GeneratePractitionerId.hbs' obj=note.act.author.assignedAuthor) as |practitionerId|}}
+                            {{>Resources/Practitioner.hbs practitioner=note.act.author.assignedAuthor ID=practitionerId.Id}},
+                            {{>References/DiagnosticReport/performer.hbs ID=(generateUUID (toJsonString note.act)) REF=(concat 'Practitioner/' practitionerId.Id)}},
+                        {{/with}}
+                        {{>Resources/Organization.hbs org=note.act.author.assignedAuthor.representedOrganization ID=(generateUUID (toJsonString note.act.author.assignedAuthor.representedOrganization))}},
+                        {{>References/DiagnosticReport/performer.hbs ID=(generateUUID (toJsonString note.act)) REF=(concat 'Organization/' (generateUUID (toJsonString note.act.author.assignedAuthor.representedOrganization)))}},
+                    {{/if}}   
+                {{/each}}
+            {{/each}}
+        {{/with}}
+        {{!-- for discharge summarries. we can onbviously refactor this or put it in its own section or just modify the getAllCdaSectionsByTemplateId to take multiple ids --}}
```

Comment:
added this. We werent picking up discharge summaries before and there were several CDAs I was looking at where they had full notes in them that we were missing